try {
  undefined.a; // Simulating the error statement
} catch (err) {
  console.error(err);
}

console.log("I am here");
